### DISCUSSION TITLE
Daily generation

### DISCUSSION THREAD

**Discussion by LueyandJoeyShow** [mletenay/home-assistant-goodwe-inverter#257] [Q&A]
Hi,
I am hoping someone can maybe help with a fix. I'm super green so go easy. 
Essentially everything works great, but my daily generation is always different than that of the Sems app.
For example right now Today's PV Generation displays as 8.7 KWh, whereas in the sems app it says 15.70 kWh.
From what i can tell the reading in HA is for the last 24h instead of today.
Can someone possibly explain to a meatball how i can fix this, I'm so close to abandoning the Sems app but this is not allowing me to give it up totally

cheers

**Comment by mletenay** [comment]
HA is taking daily from the inverter. 
It is not not last 24, it is internal counter since inverter's midnight,
Most likely, your inverter's time is off by few timezone ?
Push the Synchronize time button to set inverter internal time to HA time.
